tests/e2e/cucumber/features/file-action/passswordProtectedFolder.feature

```diff
@@ -0,0 +1,20 @@
+Feature: create folder with password
+  As a user
+  I want to create password protected folder
+  So that I can manage access to resources
+
+
+  Scenario: create a password-protected folder
+    Given "Admin" creates following user using API
+      | id    |
+      | Alice |
+    And "Alice" has logged in
+    And "Alice" creates the following resources
+      | resource | type                      | password |
+      | folder   | Password Protected Folder | %public% |
```

Comment:
better if folder name is unique so that other gets clear view what this resource is `.PasswordProtectedFolders` and `folder.psec`
```suggestion
      | sampleFolder   | Password Protected Folder | %public% |